src/utils/config.ts

```diff
@@ -66,15 +66,15 @@ export const config = {
       ABI: gravityBridge,
     },
     ETH_BTC_TREND_CELLAR: {
-      ADDRESS: "0x6b7f87279982d919Bbf85182DDeAB179B366D8f2",
+      ADDRESS: "0x6b7f87279982d919bbf85182ddeab179b366d8f2",
```

Comment:
Can you leave a comment in this file noting that addresses should be saved as lower case and that it is fine to call `.toLowerCase()` directly here.